### DISCUSSION TITLE
SkyBlock API missing stuff

### DISCUSSION THREAD

**Discussion by brunopCunha** [HypixelDev/PublicAPI#115] [Ideas]
- `/resources/skyblock/skills` Doesn't show how much is required for x level like `/resources/skyblock/collections` shows "amountRequired" for x tier.

- `/resources/skyblock/slayer` or whatever to show slayer leveling rewards like there is for skills and collections

- `/skyblock/profile` Doesn't show whether a player left the coop and when nor the reason for such.

I'm currently implementing the API in one of my projects and it would be nice if those features were added/fixed.

**Comment by puppy0cam** [comment]
Unlike collections which are recorded as a continuous milestone, skill experience is displayed as a counter that resets at each level to the player. In order to ensure that the endpoint is useful, I would recommend having one or both of the following two fields added to `collections[key].levels[arr]` objects in the `skyblock/skills` resource
* `milestoneAmountRequired` - This would be the total amount of exp required to unlock that skill level regardless of current level
* `progressionAmountRequired` - This would be the difference between the previous level and the current level.

My personal recommendation would be to implement `milestoneAmountRequired` as `/skyblock/profile` serves skill experience as the total ever without level impacting that value. It would be up to the application to subtract their current experience from their next milestone in order to obtain experience required for next level up.
